02-peer-protocol.md

```diff
@@ -1296,12 +1305,12 @@ The receiving node:
 
 #### Rationale
 
-A peer sends their `tx_signatures` as soon as they have received a valid
-`commitment_signed` message.
+A peer sends their `tx_signatures` after receiving a valid `commitment_signed`
+message, following the order specified in the [`tx_signatures` section](#the-tx_signatures-message).
 
-The channel should be preemptively closed in the case where a peer provides
-valid witness data that causes their paid feerate to fall beneath the
-`open_channel2.funding_feerate_perkw` rate. This penalizes the peer
+The channel should be preemptively double-spent in the case where a peer
+provides valid witness data that causes their paid feerate to fall beneath
+the `open_channel2.funding_feerate_perkw` rate. This penalizes the peer
```

Comment:
I'm not sure whether this actually "penalizes" the peer, especially if we're *preemptively* doing an RBF double-spend -- we end up paying extra fees.

Perhaps we should mention "productively" double spending if the opportunity arises and the transaction still hasn't confirmed.

Reply:
True, it doesn't penalize the peer, I'll rephrase that more.

Reply:
Rephrased in https://github.com/niftynei/lightning-rfc/pull/6/commits/f2daadb3d975d7165f5ce0e0060bb7f17f5cb10f, let me know if that's better.